carbon-design-system/carbon
# Update to v10 Color tokens

Closes #1513 

Update to include latest colors and themes from elements.

#### Changelog

**New**

**Changed**

- `package.json` now pulls in `@carbon/elements`
- Copy updated vendor styles into `src/globals/scss/vendor`
- Rename ibm-color variables 
- Include ibm-color variables in `_colors.scss`
- Call `@carbon/themes` in `_theme-tokens.scss`

**Removed**

#### Testing / Reviewing

- Stable is not broken
- Turn on feature flags and verify colors appear as expected
- Verify changes are only in X component styles
